docs/platforms/javascript/guides/svelte/index.mdx

```diff
@@ -83,6 +83,46 @@ const app = new App({
 export default app;
 ```
 
+```javascript {tabTitle: Svelte v5+} {filename: main.js} {"onboardingOptions": {"performance": "11, 14-21", "session-replay": "12, 24-30"}}
```

Comment:
Not sure whether to make v5 the default. [npm download stats](https://www.npmjs.com/package/svelte?activeTab=versions) definitely favor v5.

Reply:
set v5 first